src/main/java/javaserver/Routes.java

```diff
@@ -3,21 +3,38 @@
 import java.util.HashMap;
 
 public class Routes {
+	public static final String[] FILES = {"/file1", "/text-file.txt"};
+
 	private static final String[] SUPPORTED_ROOT_REQUESTS = {"GET"};
 	private static final String[] SUPPORTED_FORM_REQUESTS = {"GET", "POST", "PUT"};
 	private static final String[] SUPPORTED_METHOD_OPTIONS = {"GET", "HEAD", "POST", "OPTIONS", "PUT"};
+	private static final String[] SUPPORTED_FILE_REQUESTS = {"GET"};
+
+	private static final String[] FOUND_REDIRECT_REQUESTS = {"GET"};
```

Comment:
Yeah, that's true. Going forward I'll look into ways to configure these routes so they are more open to extension.